integration-tests/maven/src/test/java/io/quarkus/maven/it/GenerateConfigIT.java

```diff
@@ -36,16 +36,16 @@ void testAddExtensionWithASingleExtension() throws MavenInvocationException, IOE
         generateConfig("test.properties");
 
         String file = loadFile("test.properties");
-        Assertions.assertTrue(file.contains("#quarkus.log.file.enable"));
-        Assertions.assertTrue(file.contains("If file logging should be enabled"));
+        Assertions.assertTrue(file.contains("#quarkus.log.level"));
+        Assertions.assertTrue(file.contains("The default log level"));
         Assertions.assertTrue(file.contains("#quarkus.thread-pool.growth-resistance=0"));
         Assertions.assertTrue(file.contains("The executor growth resistance"));
 
         generateConfig("application.properties");
         //the existing file should not add properties that already exist
         file = loadFile("application.properties");
-        Assertions.assertTrue(file.contains("quarkus.log.file.enable=false"));
-        Assertions.assertFalse(file.contains("If file logging should be enabled"));
+        Assertions.assertTrue(file.contains("quarkus.log.level=INFO"));
+        Assertions.assertFalse(file.contains("The default log level"));
```

Comment:
I'm just wondering why this test was changed? I don't understand why it's needed so an explanation would be welcome.

Reply:
Per Baerten on Zulip: "Oh I changed them , because the structure now contains twice the value ""If file logging should be enabled"" , once for the root level, and once for configured categories. So this field was no longer a good candidate to assert on. That's why I changed it to level."